### PR TITLE
Add the relevant fix

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-7299

### Change description ###
Add the jump to feature - done similarly to other parts of application

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by DavidJayakumar — APPROVED** [review]
LGTM

File: src/staff-administrator/components/staff-add-edit-user/staff-add-edit-user-form/staff-add-edit-user-form.component.ts [hunk 1]
```diff
@@ -314,6 +314,20 @@ export class StaffAddEditUserFormComponent implements OnInit {
     }
   }
 
+  public ngAfterViewInit(): void {
+    this.fragmentFocus();
```
Comment (MateiHMCTS) [inline src/staff-administrator/components/staff-add-edit-user/staff-add-edit-user-form/staff-add-edit-user-form.component.ts:318]: Is this needed? Can't we enable anchorScrolling in routerOptions?
https://angular.io/api/router/ExtraOptions

**Review by MateiHMCTS — CHANGES_REQUESTED** [review]

File: src/staff-administrator/components/staff-add-edit-user/staff-add-edit-user-form/staff-add-edit-user-form.component.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/staff-administrator/components/staff-add-edit-user/staff-add-edit-user-form/staff-add-edit-user-form.component.ts:318]: Agree with @MateiHMCTS, @connorpgpmcelroy  Is it too much work?

**Review by udaydenduluri33 — APPROVED** [review]

File: src/staff-administrator/components/staff-add-edit-user/staff-add-edit-user-form/staff-add-edit-user-form.component.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/staff-administrator/components/staff-add-edit-user/staff-add-edit-user-form/staff-add-edit-user-form.component.ts:318]: Agreed to have a tech debt for this

Comment (MateiHMCTS) [reply src/staff-administrator/components/staff-add-edit-user/staff-add-edit-user-form/staff-add-edit-user-form.component.ts:318]: Agreed

**Review by MateiHMCTS — APPROVED** [review]

**Pull request merged**